source/documentation/common/prereqs/ref-Host_Firewall_Requirements.adoc

```diff
@@ -100,7 +100,10 @@ Client machines |{hypervisor-fullname}s
 |No
 |H14 |123 |UDP |{hypervisor-fullname}s
 
-{enterprise-linux-host-fullname}s | NTP Server |NTP requests from ports above 1023 to port 123, and responses. This port is required and open by default.
+{enterprise-linux-host-fullname}s | NTP Server |NTP requests from ports above 1023 to port 123, and responses. This port is required and open by default. |
+|H14 |4500 |TCP, UDP | {hypervisor-fullname}s | {hypervisor-fullname}s | Internet Security Protocol (IPSec)|Yes
```

Comment:
At the time of writing the bug the 14 was not there, please update it is goes one after another.